libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/TitleBar.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Stack, Text, Icon } from "@fluentui/react";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { titleBarStyles } from "./TitleBar.styles";
+export interface ITitleBarProps {
+  type: TitleBarOptions;
+}
+
+export class ITitleBarState {}
+
+enum TitleBarOptions {
+  Error,
+  Success
+}
+
+export class TitleBar extends React.Component<ITitleBarProps, ITitleBarState> {
+  public constructor(props: ITitleBarProps) {
+    super(props);
+    this.state = {};
+  }
+
+  public render(): React.ReactNode {
+    const classNames = titleBarStyles();
+
+    return (
+      <Stack
+        horizontal
+        tokens={{ childrenGap: "s1" }}
+        horizontalAlign="center"
+        verticalAlign="end"
+      >
+        <Stack.Item className={classNames.iconContainer}>
+          <Icon
+            iconName={
+              this.props.type === TitleBarOptions.Error ? "Cancel" : "Checkmark"
```

Comment:
is this iconName value seen by the user?  if it is, then we should localize it

Reply:
no, iconName is only used internally for Fluent UI's icon package